libpod/image/pull.go

```diff
@@ -18,6 +14,9 @@ import (
 	"github.com/containers/image/transports/alltransports"
 	"github.com/containers/image/types"
 	"github.com/pkg/errors"
+	"io"
```

Comment:
These go at the top, with `"fmt"` and the like

Reply:
fixed